README.md

```diff
@@ -8,24 +8,44 @@
 
 ## About
 
-This is the Git repo of the official Helm charts for YOURLS.
+This is the Git repository of the official Helm charts for YOURLS.
 
 ## Usage
 
-1. **Install Helm**  
+1. **Install Helm**
```

Comment:
Please don't remove trailing spaces on markdown.

Reply:
This was editor syntax changes but I'll get them forced back.